src/lib/shared/components/ui/Jumbotron/Jumbotron.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, BoxProps } from "@mui/material";
+import { Button, Paragraph } from '@/lib/shared/components/ui';
+
+import { styled } from '@mui/material/styles';
+import { H1 } from '@/lib/shared/components/ui';
+
+export const TEST_IDS = {
+    CONTAINER: 'container',
+} as const;
+
+export interface JumbotronProps extends BoxProps {
+    headerText: string;
+    subHeaderText?: string;
+    backgroundImageUrl?: string;
+    callToAction?: {
+        label: string;
+        onClick?: () => void;
+    }
+}
+
+const Header = styled(H1)(({ theme }) => ({
+    ...theme.typography.h4,
+}));
+
+const SubHeader = styled(Paragraph)(({ theme }) => ({
+    ...theme.typography.body1,
+}));
+
```

Comment:
I'd prefer to move these below the `Jumbotron` component just so all styled components are at the bottom of the file.